src/API/TaskService.cpp

```diff
@@ -8,21 +8,13 @@
 
 
 AddTaskResult TaskService::AddTask(const TaskDTO& task, const Task::Priority& priority){
-  auto newTask = tasksRepository_.GetTaskStorage().AddTask(task.GetTask(), priority);
-  if (newTask.has_value()) {
-    tasksRepository_.GetTaskView().AddTask(newTask.value());
-    return AddTaskResult(true);
-  }
-  return AddTaskResult(AddTaskResult::ErrorType::NOT_ENOUGH_FREE_MEMORY, false);
+  AddTaskResult addNewTask = tasksRepository_.AddTask(task, priority);
+  return addNewTask.success_;
```

Comment:
I thins these two lines can be raplaced by
return tasksRepository_.AddTask(task, priority).success_

By then it's obvious that Service is actually doing nothing.

Reply:
It ask taskRepository to add new task to storage. All TaskService does is ask another classes to do smth.
And yeah, I can merge these 2 lines in 1

Reply:
It asks taskRepository to add new task to storage. All TaskService does is ask another classes to do smth.
And yeah, I can merge these 2 lines in 1

Reply:
Also 11 line shows return type of AddTask. I think this will impruve code readabilty